internal/user/agent_test.go

```
@@ -0,0 +1,120 @@
+	}
+}
+
+func TestGetAgentsCompactByIDs(t *testing.T) {
+	mgr, db := newTestManager(t)
+	db.MustExec(`INSERT INTO users (type, email, first_name, last_name) VALUES
+		('agent', 'ada@example.com', 'Ada', 'Lovelace'),
+		('agent', 'grace@example.com', 'Grace', 'Hopper'),
+		('contact', 'customer@example.com', 'Customer', 'A')`)
+
+	var adaID, contactID int
+	db.Get(&adaID, `SELECT id FROM users WHERE email = 'ada@example.com'`)
```

golangci-lint (2.12.2)

[error] Error return value of `db.Get` is not checked

(errcheck)